var/spack/repos/jcsda-emc-bundles/packages/ufs-weather-model-env/package.py

```diff
@@ -22,22 +22,39 @@ class UfsWeatherModelEnv(BundlePackage):
         description="Build a debug version of certain dependencies (ESMF, MAPL)",
     )
     variant("python", default=True, description="Build Python dependencies")
+    variant("shared", default=True, description="Build dynamic libraries when possible")
 
-    depends_on("base-env", type="run")
     depends_on("ufs-pyenv", type="run", when="+python")
 
-    depends_on("fms@2022.01", type="run")
-    depends_on("bacio", type="run")
-    depends_on("crtm@2.4.0", type="run")
-    depends_on("g2", type="run")
-    depends_on("g2tmpl", type="run")
-    depends_on("ip", type="run")
-    depends_on("sp", type="run")
-    depends_on("w3nco", type="run")
-
-    depends_on("esmf~debug", type="run", when="~debug")
-    depends_on("esmf+debug", type="run", when="+debug")
-    depends_on("mapl~debug", type="run", when="~debug")
-    depends_on("mapl+debug", type="run", when="+debug")
-
-    # There is no need for install() since there is no code.
+    depends_on("bacio")
+    depends_on("fms")
```

Comment:
Removing the `fms` and `crtm` versions here will not work. If you install the jedi-ufs-all environment, spack may decide to use the JEDI versions of both libraries (`~release-jcsda`). The only way to install both `ufs-weather-model-env` and `jedi-fv3-env` in the same environment and get the correct versions of `fms` and `crtm` loaded for each of them is to hardcode them here.